test/integration/create_classes_test.go

```diff
@@ -0,0 +1,243 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Should we use a build tag to ensure these tests are not run by folks attempting to run unit tests?

https://mickey.dev/posts/go-build-tags-testing/

Reply:
Yes, great point! I will add that. Another safe-guard here is the way that the k8s client is configured: you have to literally drop a `kubeconfig.yaml` into the project dir, it will not just use your local configuration automatically.